### PR TITLE
Enable to alter the objective function

### PR DESCRIPTION
**Pull request by FabianHofmann**
## Changes proposed in this Pull Request
See discussion in #121 

## Checklist

- [x] Code changes are sufficiently documented; i.e. new functions contain docstrings and further explanations may be given in `doc`.
- [x] A note for the release notes `doc/release_notes.rst` of the upcoming release is included.

### CODE REVIEW

File: pypsa/components.py [hunk 1]
```diff
@@ -462,6 +462,10 @@ def lopf(self, snapshots=None, pyomo=True, solver_name="glpk",
             the model has solved and the results are extracted. It allows the user
             to extract further information about the solution, such as additional
             shadow prices.
+        skip_objective : bool, default False
+            Only taking effect when pyomo is False.
+            Skip writing the default objective function. If False, an custom
```
Comment (fneum) [inline pypsa/components.py]:
```suggestion
            Skip writing the default objective function. If False, a custom
```

File: pypsa/linopf.py [hunk 2]
```diff
@@ -831,6 +836,9 @@ def network_lopf(n, snapshots=None, solver_name="cbc",
         the model building is complete, but before it is sent to the
         solver. It allows the user to
         add/change constraints and add/change the objective function.
+    skip_objective : bool, default False
+        Skip writing the default objective function. If False, an custom
```
Comment (fneum) [inline pypsa/linopf.py]:
```suggestion
        Skip writing the default objective function. If False, a custom
```

File: pypsa/linopt.py [hunk 3]
```diff
@@ -234,6 +234,22 @@ def write_binary(n, axes):
     n.binaries_f.write(join_exprs('x' + _str_array(variables, True) + '\n'))
     return to_pandas(variables, *axes)
 
+
+def write_objective(n, terms):
+    """
+    Writer function for writing out one or mutliple objective terms.
```
Comment (fneum) [inline pypsa/linopt.py]: What does "one or multiple objective terms" mean?
```suggestion
    Writer function for writing out one or multiple objective terms.
```

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "restructuredtext.confPath": "${workspaceFolder}/doc"
+}
```
[the remark this answers is not in the document]
Comment (FabianHofmann) [reply .vscode/settings.json]: accident :)

File: pypsa/linopt.py (hunk 3 above)
Comment (FabianHofmann) [reply pypsa/linopt.py]: &rarr; one term or multiple terms

**Pull request merged**